packages/vrealize-developer-tools/.eslintrc.js

```diff
@@ -0,0 +1,147 @@
+module.exports = {
```

Comment:
Do we need a separate .eslintrc here? What's the difference with the root one?

Reply:
I haven't optimized the structure. it might not be needed at all. We clean the files futher and hoist the packages that are common for all subpackages.